Protos/V1/sessions_service.proto

```diff
@@ -10,15 +13,27 @@ import "sessions_common.proto";
 option csharp_namespace = "ArmoniK.Api.gRPC.V1.Sessions";
 
 /**
- * The SessionService provides methods for listing, getting and cancelling sessions
+ * Service for handling sessions.
  */
 service Sessions {
-  // Get all sessions with pagination
+  /**
+   * Get a sessions list using pagination, filters and sorting.
+   *
+   * Sessions are summary.
```

Comment:
That does not add any meaningful information.

Reply:
yes, will change this